### PR TITLE
Allow set up the owner of the object in S3 calls

### PR DESCRIPTION
**Pull request by pipo02mix**
Otherwise non-root users cannot open the files :)

### CODE REVIEW

File: helm/fluent-logshipping-app-chart/values.yaml [hunk 1]
```diff
@@ -41,4 +41,6 @@ fluentd:
     bucketRegion: "eu-central-1"
     bucketPathPrefix: "gs-"
     account: "00000000000" 
-    role: "my-role"
\ No newline at end of file
+    role: "my-role"
+    grant_full_control: ""
+    s3_object_key_format: "%{path}%{time_slice}_%{index}.%{file_extension}"
```
Comment (pipo02mix) [inline helm/fluent-logshipping-app-chart/values.yaml]:
```suggestion
    s3_object_key_format: "%{path}%{time_slice}_%{index}.%{file_extension}"
    
```

**Comment by pipo02mix** [comment]
Tested in TC cluster

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+## [v0.1.2] 
+
+### Added
+
+- Add optional parameter `grant_full_control` to allow define permission for the object stored in S3 bucket.
```
Comment (rossf7) [inline CHANGELOG.md]:
```suggestion
- Add optional parameter `grant_full_control` to allow defining permissions for objects stored in S3 buckets.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
...
+- Changed `s3_object_key_format` to let define the object key format via values file.
```
Comment (rossf7) [inline CHANGELOG.md]:
```suggestion
- Changed `s3_object_key_format` to allow defining the S3 object key format.
```

Can we drop the values file part?

File: helm/fluent-logshipping-app-chart/values.yaml [hunk 4]
```diff
@@ -7,8 +7,8 @@ fluentbit:
   port: 2020
   protocol: TCP
 
-  userID: 1000
-  groupID: 1000
+  userID: 0 #For systemd sshd losg it needs root
```
Comment (rossf7) [inline helm/fluent-logshipping-app-chart/values.yaml]:
```suggestion
  userID: 0 # For systemd sshd logs it needs root access
```

File: helm/fluent-logshipping-app-chart/values.yaml [hunk 5]
```diff
@@ -7,8 +7,8 @@ fluentbit:
...
+  groupID: 0 #For systemd sshd losg it needs root
```
Comment (rossf7) [inline helm/fluent-logshipping-app-chart/values.yaml]:
```suggestion
  groupID: 0 #For systemd sshd logs it needs root access
```

**Pull request merged**